### PR TITLE
Upgrade / add Gradle wrappers

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by coollog** [comment]
Hi @sschuberth thanks for your contribution! Could you share with us the commands you ran to generate these new wrappers? I'm currently seeing that the `.bat` wrappers have `LF` line endings rather than the `CRLF` required by Windows, which may cause the wrappers to not work on Windows.

**Comment by sschuberth** [comment]
@coollog, interesting that you're indeed committing `.bat` files with Windows line endings in the repository. So far my belief was it's best Git practice to always commit Unix line endings, and add a `.gitattributes` file to eventually change how files appear in the working tree. I was just running the wrapper task, but my Git for Windows installation is configured to normalize to Unix line endings on commit. So, do you want me to commit with Windows line endings, or add a `.gitattributes` file?

**Comment by coollog** [comment]
Hi @sschuberth , adding a `.gitattributes` file would be great - feel free to include `.bat` and `.cmd` files as `eol=crlf`.

**Comment by sschuberth** [comment]
@coollog, I've added new commits to this PR accordingly.

**Comment by coollog** [comment]
@sschuberth Thanks! It looks like the `gradle.properties` files got committed with CRLF line endings as well though.

File: jib-core/build.gradle [hunk 1]
```diff
@@ -1,126 +1,122 @@
...
-  def googleChecksNoJava = googleChecks.replace(fileExtensionsBefore, fileExtensionsAfter)
-  assert !googleChecks.equals(googleChecksNoJava)
-
-  config = resources.text.fromString(googleChecksNoJava)
-
-  maxErrors = 0
-  maxWarnings = 0
-}
-/* CHECKSTYLE */
+plugins {
```
Comment (coollog) [inline jib-core/build.gradle]: It looks like the `build.gradle` files got CRLF endings too.

**Comment by sschuberth** [comment]
Doh, thanks again, fixed!

**Comment by chanseokoh** [comment]
Can we merge this?

**Pull request merged**